dockstore-webservice/src/main/resources/migrations.test.xml

```diff
@@ -96,7 +96,6 @@
             <column name="mode" value="AUTO_DETECT_QUAY_TAGS_AUTOMATED_BUILDS"/>
             <column name="name" value="test1"/>
             <column name="namespace" value="test_org"/>
-            <column name="path" value="quay.io/test_org/test1"/>
```

Comment:
It seems like it would be redundant to delete these here when migrations.1.4.0 deletes the column anyway